PlayerOne/index.css

```diff
@@ -37,8 +96,88 @@ body {
     margin-left: 35%;
     margin-top: 13%;
 }
+
 #canvas {
   border: 2px solid black;
   background-color: white;
+  display: block;
+  margin: auto;
 }
 
+#canvas-controls {
+  display: flex;
+  flex-wrap: none;
+  justify-content: space-evenly;
+}
+
+div ~ ul {
+  list-style-type: none;
+}
+
+:not(button) {
+  color: white;
+  font-family: "Questrial", "sans-serif";
+}
+
+div > button {
+  background-color: #FFE39A;
+  color: black;
+  padding: 10px;
+  border: none;
+  text-decoration: none;
+}
+
+div {
+  padding :10px;
+}
+
+p:first-child {
+  font-size: 20px;
+}
+
+@keyframes loading {
+  0% {
+    /* background-image: linear-gradient(to right, #FFE39A, blue); */
+    background-position: 100% 50%;
+  }
+
+  100% {
+    background-position: 0% 50%;
+    /* background-image: linear-gradient(to right, blue, #FFE39A,); */
+  }
+}
+
+@keyframes spin {
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+h1:empty {
+  display: block;
+  margin: 0 auto;
+
+  width: 3rem;
+  height: 3rem;
+  border: 7px solid rgba(255, 255, 255, 0.5);
+  background-color: rgba(255, 255, 255, 0.3);;
+  /* border-top-color: #fff;
+  border-radius: ; */
+  animation: 1s spin infinite ease-in-out;
+}
```

Comment:
when a <h1> tag is empty (e.g. when we're waiting for a random word) it will show a loading animation.